### DISCUSSION TITLE
sardine.delete(url) does not return after second call

### DISCUSSION THREAD

**Discussion by DannyAH** [lookfirst/sardine#410] [Q&A]
Hi,
we using sardine to delete imported files into our software remote on a transfer-server.
For a couple of days now, sardine.delete wont come back, after calling the delete a second time for a different file. The first time called it is working like a charm.

I hope you could help me.
Kind Regards,
Danny

**Comment by woody81** [comment]
We are facing the same issue. When we call delete to webDav for the second time on the same connection, Tomcat got stuck and returns 503 after some time. No exception anywhere and the file is still in WebDav. If I reset a connection after each delete, then it works. We use the latest version and Tomcat ver. 7.0.92

**Comment by jdickel** [comment]
Same issue here. Our Docker container is stuck without any Exception. Same for the local setup.
Have you solved that issue or did you find a workaround?
